lldb/test/Shell/Commands/command-disassemble.s

```diff
@@ -82,20 +82,25 @@
 # CHECK-NEXT: (lldb) disassemble --name case2
 # CHECK-NEXT: command-disassemble.s.tmp`n1::case2:
 # CHECK-NEXT: command-disassemble.s.tmp[0x2044] <+0>: int    $0x32
-# CHECK-NEXT: warning: Not disassembling a range because it is very large [0x0000000000002046-0x0000000000004046). To disassemble specify an instruction count limit, start/stop addresses or use the --force option.
+# CHECK-NEXT: warning: Not disassembling a function because it is very large [0x0000000000002046-0x0000000000004046). To disassemble specify an instruction count limit, start/stop addresses or use the --force option.
 # CHECK-NEXT: (lldb) disassemble --name case3
-# CHECK-NEXT: error: Not disassembling a range because it is very large [0x0000000000004046-0x0000000000006046). To disassemble specify an instruction count limit, start/stop addresses or use the --force option.
-# CHECK-NEXT: Not disassembling a range because it is very large [0x0000000000006046-0x0000000000008046). To disassemble specify an instruction count limit, start/stop addresses or use the --force option.
+# CHECK-NEXT: error: Not disassembling a function because it is very large [0x0000000000006046-0x0000000000007046)[0x0000000000009046-0x000000000000a046). To disassemble specify an instruction count limit, start/stop addresses or use the --force option.
```

Comment:
If you specify a limit does it start from the lowest address of the function or the entry point. I'd guess lowest address, because there's no way to say "100 forward and 100 backwards".

Reply:
But then does that make it not useful, because it won't start at the prologue?

Do we have any idea what order the chunks are in? I guess not, because in theory it could jump between them at runtime, maybe even revisit them.

Reply:
Currently it will print  `limit` instructions from every range (this is shown in the next test case, line 89). This is mainly a side-effect of the fact that we don't make much of a difference between two ranges belonging to the same function and two ranges from two different functions, but maybe it also kind of slightly makes sense? At least it avoids us needing to worry about which range will contain the entry point (the ranges are sorted so it might not be the first one)

Reply:
All the options are a bit strange but yes that achieves the objective of not spamming the terminal in a way where it would be hard to tell where the output begins and ends. Which is especially important if the function spans multiple ranges.